### PR TITLE
feat: Added docker and docker-compose file

### PR DESCRIPTION
**Pull request by Rajat117**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,14 +121,18 @@ A boilerplate for [Node.js](https://nodejs.org/en) App.
...
 git clone https://github.com/GeekyAnts/express-typescript.git;
 
 # Goto the cloned project folder.
 cd nodets;
+```
+
+```bash
+# Without Docker
+
+> Note: It is preassumed here that you have mongoose running in background & you have created the database.
```
Comment (faizahmedfarooqui) [inline README.md:135]: @Rajat117 Please convert this line into a comment.

File: README.md [hunk 2]
```diff
@@ -144,6 +148,19 @@ vim .env;
 npm run dev;
 ```
 
+```bash
+# With Docker
+
+> Note: It is preassumed here that you have docker running in background
```
Comment (faizahmedfarooqui) [inline README.md:154]: @Rajat117 Please convert this line into comment.

**Review by faizahmedfarooqui — COMMENTED** [review]
@Rajat117 Please go through 2 comments I've added in this review. Apart from that eveything else looks good to me.

**Comment by Rajat117** [comment]
@faizahmedfarooqui I've made the advised changes. Please have a look

**Comment by faizahmedfarooqui** [comment]
@Rajat117 Everything looks perfect. Squashed & merge!

**Pull request merged**